### DISCUSSION TITLE
Audio Master output?

### DISCUSSION THREAD

**Discussion by scwfan06** [obsproject/obs-studio#4430] [Development Questions] [answered]
It would be nice to be able to select master audio output device.
I want to stream via an external encoder and I want to send the stream via HDMI but I cannot send audio except if I set it as monitor device which isnt very optimal.
Thanks

**Comment by Fenrirthviti** [comment, marked as answer]
https://ideas.obsproject.com/posts/170/output-volume-visualization-master-mix

Please use the ideas page for feature discussions
